ep/lh/ep-lh-0018.md

```diff
@@ -4,29 +4,77 @@
+| Authors      | YankaPt, svoboda-rabstvo                |
 | Status       | Draft                                   |
+
 ## Abstract
+
 This document describes the structure of engine output.
 
+## Motivation
+
+lhexec [`ep-lh-0014`](#related) work with many engines. To use each engine need to unify output of engine. This enhancement will define how to unify output engine.
+
 ## Proposal
+
 The output of each engine should is an array. Each item of array is described by:
+
 - `path` - The path relative to the analysis root
 - `messages` - List of messages in the path
 
 ## Messages
-Messages is an array. Each item is a `message`. `message` is a JSON Schema object with the following properties:
+
+Messages is a array of errors. Each `message` describe error and where is he located, and represent a JSON object with the following properties:
 
 | Property    | Type     | Required | Description |
 | -           | :-:      | :-:      | -           |
-| message          | string   | +        | The short description of the message |
-| description      | string   | -        | The explanatory text of the message  |
-| severity     | string      | +        | The severity of the message: “verbose”, “hint”, “information”, “warning” or “error” |
-| line        | integer    | +        | The line where the message is located|
-| lineEnd         | integer   | -        | The end line where the message is located (the same as line by default)|
-| column          | integer   | -        | The column where the message is located |
-| columnEnd          | integer   | -        | The end column where the message is located |
-| ruleId          | string   | -        | The id of the rule that produced the message |
-| ruleName          | string   | -        | The name of the rule that produced the message |
-| ruleNs          | string   | -        | The namespace of the rule that produced the message |
+| message     | string   | +        | The short description of the message |
+| description | string   | -        | The explanatory text of the message  |
+| severity    | string   | +        | The severity of the message: “verbose”, “hint”, “information”, “warning” or “error” |
+| line        | integer  | +        | The line where the message is located|
+| lineEnd     | integer  | -        | The end line where the message is located (the same as line by default)|
+| column      | integer  | -        | The column where the message is located |
+| columnEnd   | integer  | -        | The end column where the message is located |
+| ruleId      | string   | -        | The id of the rule that produced the message |
+| ruleName    | string   | -        | The name of the rule that produced the message |
+| ruleNs      | string   | -        | The namespace of the rule that produced the message |
+
+## Example
+
+Here is the full example for structure of engine output:
+
+```json
+[
+    {
+        "path": "pathToFile",
```

Comment:
JSON is not formatted, it's minor but could you please use the same format as in the schema?